### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by pastorp3**
## Calculator Milestone 2: Components

In this milestone I:
- Added Components folder
- Added Display component
- Added Button Panel component
- Added Button component
- Added prop-types for each component

### CODE REVIEW

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable  react/require-default-props */
```
Comment (descholar-ceo) [inline src/components/Button/Button.js]: [required] don't disable this rule please, enable it and fix the linters errors. That will help you to master dealing with eslint common errors

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @pastorp3  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your project it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implement all of the minimal requirements of this project, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] I noticed that you are disabling a lot of eslint rules, I would suggest you enable them back and try to fix them so that you will become very familiar with fixing eslint errors which will make your codes very clean and following starndard convetions
- [x] I noticed again you have created extra directories for `src/components/button/button.js`, `src/components/buttonPanel/ButtonPanel.js`, `src/display/components/Display.js` => That was not needed at all, you could put `Button.js`, `Display.js`, and `ButtonPanel.js` directly in `src/components/` directory.
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Comment by pastorp3** [comment]
- Issue with linters fixed
- I decided to keep the extra folders for components because I can have a better structure and have the css file and js file in the same folder

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @pastorp3 :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**